1.0/nanoserver/sdk/Dockerfile

```diff
@@ -4,7 +4,7 @@ FROM microsoft/dotnet:1.0.4-sdk-nanoserver-10.0.14393.953
 # Install node
 ENV NODE_VERSION 6.10.2
 
-RUN Invoke-WebRequest https://nodejs.org/dist/v${env:NODE_VERSION}/node-v${env:NODE_VERSION}-win-x64.zip -outfile node.zip; `
+RUN Invoke-WebRequest -UseBasicParsing https://nodejs.org/dist/v${env:NODE_VERSION}/node-v${env:NODE_VERSION}-win-x64.zip -outfile node.zip; `
```

Comment:
Interesting. This works when building the images on Windows 10 Pro Creator Update. Is this a known issue or should we open a bug?